File: CS-Examples/05_Convertors/ToImage/ToImage.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Doc;
using Spire.Doc.Documents;

namespace ConvertToImage
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            //Create word document
            Document document = new Document();
            document.LoadFromFile(@"..\..\..\..\..\..\Data\Word.doc");

            //Save doc file.
            Image img = document.SaveToImages(0, ImageType.Metafile);
            img.Save("sample.bmp", System.Drawing.Imaging.ImageFormat.Bmp);

            //Launching the image file.
            WordDocViewer("sample.bmp");
        }

        private void WordDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB-Examples/05_Convertors/ToImage/ToImage.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Doc
Imports Spire.Doc.Documents

Namespace ConvertToImage
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			'Create word document
            Dim document_Renamed As New Document()
            document_Renamed.LoadFromFile("..\..\..\..\..\..\Data\Word.doc")

			'Save doc file.
            Dim img As Image = document_Renamed.SaveToImages(0, ImageType.Metafile)
            img.Save("sample.bmp", System.Drawing.Imaging.ImageFormat.Bmp)

			'Launching the image file.
			WordDocViewer("sample.bmp")
		End Sub

		Private Sub WordDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace
